### PR TITLE
[FIX] Redirect to default URL after 1FA when default policy is one_factor.

### LINKED ISSUE #581: Default redirection not working
First of all: thanks for all the work. I started using it recently and it has been working great!
 
However, I noted that the default_redirection_url is not working correctly. When I go to my Authelia URL (i.e. auth.example.com) and after succesfull login (even 2FA), I do not get redirected to the default redirection URL, while I have this configurated. Am I doing something wrong here, or is this a minor bug?

```
# Default redirection URL
# If user tries to authenticate without any referer, Authelia
# does not know where to redirect the user to at the end of the
# authentication process.
# This parameter allows you to specify the default redirection
# URL Authelia will use in such a case.
# Note: this parameter is optional. If not provided, user won't
# be redirected upon successful authentication.
default_redirection_url: https://home.example.com

```

### PR DESCRIPTION
**Pull request by clems4ever**
User is now redirected to the default redirection URL after 1FA if
the default policy is set to one_factor and there is no target URL
or if the target URL is unsafe.

Also, if the default policy is set to one_factor and the user is already
authenticated, if she visits the login portal, the 'already authenticated'
view is displayed with a logout button.

This fixes #581.

### CODE REVIEW

**Comment by nightah** [comment]
Looks like one of the tests is still failing: https://buildkite.com/authelia/authelia/builds/331#1e895873-0ec2-4ce1-b6cf-1e710a78f2a7

File: internal/authorization/authorizer_test.go [hunk 1]
```diff
@@ -255,6 +255,15 @@ func (s *AuthorizerSuite) TestShouldCheckResourceMatching() {
 	tester.CheckAuthorizations(s.T(), John, "https://resource.example.com/xyz/embedded/abc", Bypass)
 }
 
+func (s *AuthorizerSuite) TestPolicyToLevel() {
+	s.Assert().Equal(Bypass, PolicyToLevel("bypass"))
+	s.Assert().Equal(OneFactor, PolicyToLevel("one_factor"))
+	s.Assert().Equal(TwoFactor, PolicyToLevel("two_factor"))
+	s.Assert().Equal(Denied, PolicyToLevel("deny"))
+
+	s.Assert().Equal(Denied, PolicyToLevel("whatever"))
```
Comment (nightah) [inline internal/authorization/authorizer_test.go]: What is this used for? I can't seem to see any other references.

Comment (clems4ever) [reply internal/authorization/authorizer_test.go]: It's a unit test to make sure the behavior of the transform function is covered. Not a big added value but I wanted to freeze the behavior of the undetected policy (set it to deny).

**Comment by clems4ever** [comment]
> Looks like one of the tests is still failing: https://buildkite.com/authelia/authelia/builds/331#1e895873-0ec2-4ce1-b6cf-1e710a78f2a7

Yes, I'm on it.

**Comment by clems4ever** [comment]
It's weird, this suite works all fine on my laptop. Maybe an issue with timeouts. I'll check that.

File: internal/suites/Standalone/users.yml [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
     email: harry.potter@authelia.com
     groups: []
   james:
-    password: '$6$rounds=500000$jgiCMRyGXzoqpxS3$w2pJeZnnH8bwW3zzvoMWtTRfQYsHbWbD/hquuQ5vUeIyl9gdwBIt6RWk2S6afBA0DPakbeWgD/4SZPiS0hYtU/'
+    password: $6$rounds=500000$jgiCMRyGXzoqpxS3$w2pJeZnnH8bwW3zzvoMWtTRfQYsHbWbD/hquuQ5vUeIyl9gdwBIt6RWk2S6afBA0DPakbeWgD/4SZPiS0hYtU/
     email: james.dean@authelia.com
     groups: []
   john:
-    password: '$6$rounds=50000$LnfgDsc2WD8F2qNf$0gcCt8jlqAGZRv2ee3mCFsfAr1P4N7kESWEf36Xtw6OjkhAcQuGVOBHXp0lFuZbppa7YlgHk3VD28aSQu9U9S1'
+    password: '{CRYPT}$6$rounds=50000$LnfgDsc2WD8F2qNf$0gcCt8jlqAGZRv2ee3mCFsfAr1P4N7kESWEf36Xtw6OjkhAcQuGVOBHXp0lFuZbppa7YlgHk3VD28aSQu9U9S1'
```
Comment (james-d-elliott) [inline internal/suites/Standalone/users.yml]: This seems inconsistent with every other users.yml. The others are double-ticks and don't include {CRYPT}. Is this intentional?

Comment (clems4ever) [reply internal/suites/Standalone/users.yml]: nop, it's not intentional.

**Pull request merged**